java/samples/sample-code/src/main/java/com/microsoft/semantickernel/samples/Config.java

```diff
@@ -25,7 +26,7 @@ public class Config {
 
     private static final List<String> DEFAULT_PROPERTIES_LOCATIONS = Arrays.asList(
             "java/samples/conf.properties",
-            System.getProperty("user.home") + "/.sk/conf.properties"
+            System.getProperty("user.home") + File.separator + ".sk" + File.separator + "conf.properties"
```

Comment:
I also tried putting it in ~/.sk/conf.properties - gives same exception.

Reply:
Hey @maxandersen , did you fix this?

Reply:
Yeah. It was just the missing org key.